File: code-examples/php/collation/findOneAndDelete-collation.php
<?php
// Create the connection
$manager = new MongoDB\Driver\Manager("mongodb://localhost:27017");

// Specify the filter criteria
$filter = ['a' => ['$gt' => 100]];

// Specify the options
$options = ['collation' => ['locale' => 'en', 'numericOrdering' => true]];

// Create and execute the command
$bulk = new MongoDB\Driver\BulkWrite();
$bulk->delete($filter, $options);

$manager->executeBulkWrite('dbname.myColl', $bulk);
?>

File: code-examples/scala/collation/findOneAndDelete-collation.scala
import org.mongodb.scala.model.Filters._
import org.mongodb.scala.model.Collation
import org.mongodb.scala.MongoCollection
import org.bson.Document

val myColl: MongoCollection[Document] = ???

val collation = Collation.builder().locale("en").numericOrdering(true).build()

myColl.findOneAndDelete(gt("a", "100")).collation(collation).subscribe()
